AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -205,13 +205,43 @@ def deferred_fog(property: str = 'name') -> str:
           - 'requires' a list of component names as strings required by this component.
             Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.\n
           - 'Enable Deferred Fog' Toggle active state of the component True/False
+          - 'Fog Color' Sets the fog color. RGB value set using azlmbr.math.Vector3 tuple.
+          - 'Fog Start Distance' Distance from the viewer where the fog starts (0.0, 5000.0).
+          - 'Fog End Distance' Distance from the viewer where fog masks the background scene (0.0, 5000.0).
+          - 'Fog Bottom Height' Height at which the fog layer starts (-5000.0, 5000.0).
+          - 'Fog Max Height' Height of the fog layer top (-5000.0, 5000.0).
+          - 'Noise Texture' The noise texture used for creating the fog turbulence.
+          - 'Noise Texture First Octave Scale' Scale of the first noise octave (INF, INF).
+            Higher values indicates higher frequency. Values set using azlmbr.math.Vector2 tuple.
+          - 'Noise Texture First Octave Velocity' Velocity of the first noise octave UV coordinates (INF, INF).
+            Values set using azlmbr.math.Vector2 tuple.
+          - 'Noise Texture Second Octave Scale' Scale of the second noise octave (INF, INF).
+            Higher values indicates higher frequency. Values set using azlmbr.math.Vector2 tuple.
+          - 'Noise Texture Second Octave Velocity' Velocity of the second noise octave UV coordinates (INF, INF).
+            Values set using azlmbr.math.Vector2 tuple.
+          - 'Octaves Blend Factor' Blend factor between the noise octaves (0.0, 1.0).
+          - 'Enable Turbulence Properties' Enables Turbulence Properties.
+          - 'Enable Fog Layer' Enables the fog layer.
```

Comment:
nit: I'm not sure how 'Enable Fog Layer' is different than 'Enable Deferred Fog'

Reply:
Enable deferred fog is the toggle for the active state of the component. When it's off, nothing else on the component registers in the editor.

The Enable Fog Layer is an actual layer of fog applied to the scene as opposed to just general fogginess throughout the entire scene. That's the best description I can give it anyway.